dev/otel-agent-config.yaml

```diff
@@ -1,10 +1,16 @@
 receivers:
   jaeger:
     protocols:
+      thrift_compact: # on port 6831
```

Comment:
what is the purpose of exposing an empty object?

Reply:
Empty adds the default listener.
Its equivalent to the following structure:
```
thrift_compact:
  endpoint: 0.0.0.0:6831
```